splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/CachedOperation.java

```diff
@@ -134,6 +109,8 @@ public String prettyPrint(int indentLevel) {
     public DataSet<ExecRow> getDataSet(DataSetProcessor dsp) throws StandardException {
         if (!isOpen)
             throw new IllegalStateException("Operation is not open");
+        populateCache();
+        source.openCore(dsp);
```

Comment:
`source.openCore();` is already called once in `populateCache()`, is it necessary to call it again here?

Reply:
It it closed by populateCache() eventually